plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -532,6 +559,20 @@ public Optional<CatalogSchemaTableName> redirectTable(ConnectorSession session,
     @Override
     public LocatedTableHandle getTableHandle(ConnectorSession session, SchemaTableName tableName)
     {
+        throw new UnsupportedOperationException("This method is not supported because getTableHandle with versions is implemented instead");
```

Comment:
nit: `Calling this method is not supported.  getTableHandle with versions is implemented.`